src/Approvals/SFA.DAS.Approvals/Application/BulkUpload/Commands/BulkUploadUpdateLogCommand.cs

```diff
@@ -0,0 +1,10 @@
+﻿using MediatR;
+
+namespace SFA.DAS.Approvals.Application.BulkUpload.Commands
+{
+    public class BulkUploadUpdateLogCommand : IRequest<BulkUploadUpdateLogResult>
+    {
+        public long? LogId { get; set; }
+        public long? ProviderId { get; set; }
+    }
+}
```

Comment:
sorry, see previous comments about unused code

Reply:
hi paul, removed this as part of removing all Update endpoint